CMakeLists.txt

```diff
@@ -541,6 +545,14 @@ if("${SWIFT_NATIVE_SWIFT_TOOLS_PATH}" STREQUAL "")
   set(SWIFT_NATIVE_SWIFT_TOOLS_PATH "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
 endif()
 
+if(LIBSWIFT_BUILD_MODE STREQUAL "HOSTTOOLS" AND NOT HOST_SWIFT_EXEC)
+  find_program(HOST_SWIFT_EXEC "swiftc"
```

Comment:
Can you use https://cmake.org/cmake/help/latest/module/CheckLanguage.html#module:CheckLanguage and enable Swift just as a normal cmake language?